scout/parse/variant/genotype.py

```diff
@@ -324,24 +326,23 @@ def get_ref_depth(
     """Get reference read depth"""
     ref_depth = int(variant.gt_ref_depths[pos])
     if ref_depth == -1:
-        if any([paired_end_ref, split_read_ref]):
-            ref_depth = 0
-            if paired_end_ref:
-                ref_depth += paired_end_ref
-            if split_read_ref:
-                ref_depth += split_read_ref
-
-        if any([spanning_ref, flanking_ref, inrepeat_ref]):
-            ref_depth = 0
-            if spanning_ref:
-                ref_depth += spanning_ref
-            if flanking_ref:
-                ref_depth += flanking_ref
-            if inrepeat_ref:
-                ref_depth += inrepeat_ref
-
-        if spanning_mei_ref:
-            ref_depth += spanning_mei_ref
+        return
+
+    REF_ITEMS_LIST: List[tuple] = [
+        (paired_end_ref, split_read_ref),
+        (spanning_ref, flanking_ref, inrepeat_ref),
+    ]
+
+    for list in REF_ITEMS_LIST:
+        if all(item is None for item in list) is True:
+            continue
+        alt_depth = 0
```

Comment:
```suggestion
        ref_depth = 0
```

Reply:
Ah, right, thanks. I'll write a test, it's a good idea!